### PR TITLE
Synapse 1.48.0

### PR DESCRIPTION
**Pull request by babolivier**
(afranke: Putting a non empty description so the netlify preview link gets added)

Preview: https://pr1130--matrix-org-previews.netlify.app

### CODE REVIEW

File: gatsby/content/blog/2021/11/2021-11-30-synapse-1.48.0-released.mdx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## New [admin APIs](https://matrix-org.github.io/synapse/latest/usage/administration/admin_api/index.html) and more compatibility with the Matrix specification
+
+This release also introduces a handful of new admin APIs, allowing administrators to [un-shadow-ban users](https://matrix-org.github.io/synapse/latest/admin_api/user_admin_api.html#controlling-whether-a-user-is-shadow-banned), [block a room](https://matrix-org.github.io/synapse/latest/admin_api/rooms.html#block-room-api), and [run specific background updates](https://matrix-org.github.io/synapse/latest/usage/administration/admin_api/background_updates.html#run) (but we'll talk about this last one a bit later on). The [delete room API](https://matrix-org.github.io/synapse/latest/admin_api/rooms.html#delete-room-api) has also been updated to be able to run in the background or to block a room pre-emptively, even if the server doesn't know about it yet.
+
+This release also brings Synapse closer to compatiblity with [Matrix v1.1](https://matrix.org/blog/2021/11/09/matrix-v-1-1-release) by adding support for the `/_matrix/client/v3` and `/_matrix/media/v3` API from the latest version of the specification.
```
Comment (aaronraimist) [inline gatsby/content/blog/2021/11/2021-11-30-synapse-1.48.0-released.mdx:21]:
```suggestion
This release also brings Synapse closer to compatiblity with [Matrix v1.1](https://matrix.org/blog/2021/11/09/matrix-v-1-1-release) by adding support for the `/_matrix/client/v3` and `/_matrix/media/v3` APIs from the latest version of the specification.
```

**Comment by callahad** [comment]
I think we should say something up top about the impending Py3.6/Pg9.6 deprecations, and there are a few small tweaks to verbiage I'd like to make, but otherwise looks great! I'll get those in and push the post early in the morning.

File: gatsby/content/blog/2021/11/2021-11-30-synapse-1.48.0-released.mdx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+date: '2021-11-30'
+title: Synapse 1.48.0 released
+categories:
+- Releases
+author: Brendan Abolivier
+---
+
+[Synapse 1.48.0](https://github.com/matrix-org/synapse/releases/tag/v1.48.0) is out now!
+
```
Comment (callahad) [inline gatsby/content/blog/2021/11/2021-11-30-synapse-1.48.0-released.mdx:10]:
```suggestion
> __NOTE:__ Synapse 1.49, due out on December 14th, will be the **last release of Synapse to support Python 3.6 or PostgreSQL 9.6** per our [platform dependency deprecation policy](https://matrix-org.github.io/synapse/v1.48/deprecation_policy.html). Accordingly, we will remove support for Ubuntu 18.04 LTS (Bionic) at the same date, as it ships with Python 3.6.

```

File: gatsby/content/blog/2021/11/2021-11-30-synapse-1.48.0-released.mdx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+## Password resets and identity servers
+
+This release removes the long-deprecated `trust_identity_server_for_password_resets` configuration option. This option was initially deprecated in [Synapse 1.4.0](https://github.com/matrix-org/synapse/releases/tag/v1.4.0) back in October 2019.
+
+Admins of servers still using this configuration option will need to update their Synapse configuration to send password resets through an SMTP server directly rather than relying on identity servers to send them on their behalf.
+
+## New [admin APIs](https://matrix-org.github.io/synapse/latest/usage/administration/admin_api/index.html) and more compatibility with the Matrix specification
```
Comment (callahad) [inline gatsby/content/blog/2021/11/2021-11-30-synapse-1.48.0-released.mdx:17]:
```suggestion
## New [admin APIs](https://matrix-org.github.io/synapse/latest/usage/administration/admin_api/index.html) and improved alignment with Matrix 1.1
```

File: gatsby/content/blog/2021/11/2021-11-30-synapse-1.48.0-released.mdx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+Lately the Synapse team has been doing some work to improve the performance of these background updates. More specifically, this release includes a [performance fix](https://github.com/matrix-org/synapse/pull/11421) for a background update introduced in Synapse 1.47.0, as well as a [new admin API](https://matrix-org.github.io/synapse/latest/usage/administration/admin_api/background_updates.html#run) to let admins rerun specific updates.
+
+Future Synapse updates will also include [module capabilities](https://github.com/matrix-org/synapse/pull/11306) and [more configuration options](https://github.com/matrix-org/synapse/issues/11260) for controlling background updates.
+
+## Everything else
+
+This release also includes some improved support of [MSC3440](https://github.com/matrix-org/matrix-doc/pull/3440) to help threading. It also adds support for the stable identifiers from [MSC2778](https://github.com/matrix-org/matrix-doc/pull/2778), bringing Synapse closer to supporting end-to-end (or end-to-bridge) encryption support for application services.
+
```
Comment (callahad) [inline gatsby/content/blog/2021/11/2021-11-30-synapse-1.48.0-released.mdx:34]:
```suggestion

We also now publish a Docker image, `matrixdotorg/synapse:develop`, which tracks the development head of Synapse.

```

File: gatsby/content/blog/2021/11/2021-11-30-synapse-1.48.0-released.mdx [hunk 5]
```diff
@@ -0,0 +1,40 @@
+---
+date: '2021-11-30'
+title: Synapse 1.48.0 released
+categories:
+- Releases
+author: Brendan Abolivier
+---
+
+[Synapse 1.48.0](https://github.com/matrix-org/synapse/releases/tag/v1.48.0) is out now!
+> __NOTE:__ Synapse 1.49, due out on December 14th, will be the **last release of Synapse to support Python 3.6 or PostgreSQL 9.6** per our [platform dependency deprecation policy](https://matrix-org.github.io/synapse/v1.48/deprecation_policy.html). Accordingly, we will remove support for Ubuntu 18.04 LTS (Bionic) at the same date, as it ships with Python 3.6.
```
Comment (babolivier) [inline gatsby/content/blog/2021/11/2021-11-30-synapse-1.48.0-released.mdx:10]:
```suggestion
[Synapse 1.48.0](https://github.com/matrix-org/synapse/releases/tag/v1.48.0) is out now!

> __NOTE:__ Synapse 1.49, due out on December 14th, will be the **last release of Synapse to support Python 3.6 or PostgreSQL 9.6** per our [platform dependency deprecation policy](https://matrix-org.github.io/synapse/v1.48/deprecation_policy.html). Accordingly, we will remove support for Ubuntu 18.04 LTS (Bionic) at the same date, as it ships with Python 3.6.
```

Comment (callahad) [inline gatsby/content/blog/2021/11/2021-11-30-synapse-1.48.0-released.mdx:11]: Hm. GitHub dropped a newline here. Let's add it back.

```suggestion

> __NOTE:__ Synapse 1.49, due out on December 14th, will be the **last release of Synapse to support Python 3.6 or PostgreSQL 9.6** per our [platform dependency deprecation policy](https://matrix-org.github.io/synapse/v1.48/deprecation_policy.html). Accordingly, we will remove support for Ubuntu 18.04 LTS (Bionic) at the same date, as it ships with Python 3.6.
```

**Review by callahad — APPROVED** [review]
LGTM! Thank you so much, and apologies on delayed review.

File: gatsby/content/blog/2021/11/2021-11-30-synapse-1.48.0-released.mdx (hunk 5 above)
Comment (babolivier) [reply gatsby/content/blog/2021/11/2021-11-30-synapse-1.48.0-released.mdx:11]: Way ahead of you :stuck_out_tongue: https://github.com/matrix-org/matrix.org/pull/1130#discussion_r760370938

**Pull request merged**